packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/Palette.tsx

```diff
@@ -282,12 +287,38 @@ export const Palette = ({ diagramElementId, onDirectEditClick }: PaletteProps) =
     }
   };
 
+  const invokeFadeDiagramElementTool = () => {
+    fadeDiagramElements([diagramElementId], true);
+  };
+
+  const invokeHideDiagramElementTool = () => {
+    hideDiagramElements([diagramElementId], true);
+  };
+
   return (
     <NodeToolbar className={classes.toolbar} position={Position.Top}>
       <div className={classes.toolEntries}>
         {tools.filter(isSingleClickOnDiagramElementTool).map((tool) => (
           <Tool tool={tool} onClick={handleToolClick} key={tool.id} />
         ))}
+        <IconButton
```

Comment:
Using Icon buttons make svg icon irrelevant.

Reply:
These button will always be available, event in the diagram palette.
We do not want these buttons in the diagram palette